src/deploy/osp_deployer/settings/sample_csp_profile.ini

```diff
@@ -620,3 +620,56 @@ node_types=
 # dhcp_end = 192.168.122.250
 # inspection_iprange = 192.168.122.51,192.168.122.120
 # gateway = 192.168.122.1
+
+[PowerFlex Settings]
+
+# Settings for PowerFlex OS Storage Backend
+enable_powerflex_backend = false
+
+# Method to retrieve RPMs before Installation begins
+# Can be either local or external
+powerflex_rpms_method = local
+
+# Used to override default RPMs path.
+# PowerFlexRpmsPath = '/opt/dellemc/powerflex/rpms'
+
+# Name of the PowerFlex OS cluster
+powerflex_cluster_name = powerflex
+
+# Name of the  PowerFlex OS Protection Domain
+# A single protection is defined for the whole system.
+powerflex_protection_domain = domain1
```

Comment:
Dont think there should be spaces in all of those settings , regardless of them being picked up within the String values (which would cause issues), they should be formatted the same way as the rest of the settings

Reply:
> Dont think there should be spaces in all of those settings , regardless of them being picked up within the String values (which would cause issues), they should be formatted the same way as the rest of the settings

Done